packages/documentation-framework/components/sectionGallery/sectionDataListLayout.js

```diff
@@ -55,10 +55,10 @@ export const SectionDataListLayout = ({
                       </DataListCell>
                     ),
                     <DataListCell width={5} key="text-description">
-                      <Split className={hasListText ? 'pf-v6-u-mb-md' : null}>
+                      <Split className={hasListText ? "pf-v6-u-mb-md" : null}>
                         <SplitItem isFilled>
-                          <Content>
-                            <Content component={ContentVariants.h2}>
+                          <Content isEditorial>
+                            <Content component={ContentVariants.h2} isEditorial>
```

Comment:
Same comment as before - you should only need `isEditorial` on one of those. If it's on the parent `<Content isEditorial>` then any content component children will get editorial styling without having to use `isEditorial` on the children specifically. Since there is only one child, as long as you want to keep the wrapping `<Content />` (removing it would add a margin to the `h2` if there is an element in the DOM that comes after the `h2`), it wouldn't matter which element has `isEditorial`